consortia/argocd-app-templates/appsetup-stable.yaml

```diff
@@ -23,19 +23,232 @@ metadata:
+                  nginx.ingress.kubernetes.io/cors-allow-origin: "http://localhost:3000, https://*.stable.demo.catena-x.net"
+                tls:
+                  - secretName: "tls-secret"
+                    hosts:
+                      - "portal.stable.demo.catena-x.net"
+                hosts:
+                  - host: "portal.stable.demo.catena-x.net"
+                    paths:
+                      - path: "/(.*)"
+                        pathType: "Prefix"
+                        backend:
+                          service: "portal"
+                          port: 8080
+                      - path: "/registration/(.*)"
+                        pathType: "Prefix"
+                        backend:
+                          service: "registration"
+                          port: 8080
+                      - path: "/((assets|documentation)/.*)"
+                        pathType: "Prefix"
+                        backend:
+                          service: "assets"
+                          port: 8080
+            backend:
+              ingress:
+                enabled: true
+                ingressClassName: "nginx"
+                annotations:
+                  nginx.ingress.kubernetes.io/use-regex: "true"
+                  nginx.ingress.kubernetes.io/enable-cors: "true"
+                  nginx.ingress.kubernetes.io/proxy-body-size: "8m"
+                  nginx.ingress.kubernetes.io/cors-allow-origin: "http://localhost:3000, https://*.stable.demo.catena-x.net"
+                tls:
+                  - secretName: "tls-secret"
+                    hosts:
+                      - "portal-backend.stable.demo.catena-x.net"
+                hosts:
+                  - host: "portal-backend.stable.demo.catena-x.net"
+                    paths:
+                      - path: "/api/registration"
+                        pathType: "Prefix"
+                        backend:
+                          service: "registration-service"
+                          port: 8080
+                      - path: "/api/administration"
+                        pathType: "Prefix"
+                        backend:
+                          service: "administration-service"
+                          port: 8080
+                      - path: "/api/notification"
+                        pathType: "Prefix"
+                        backend:
+                          service: "notification-service"
+                          port: 8080
+                      - path: "/api/apps"
+                        pathType: "Prefix"
+                        backend:
+                          service: "marketplace-app-service"
+                          port: 8080
+                      - path: "/api/services"
+                        pathType: "Prefix"
+                        backend:
+                          service: "services-service"
+                          port: 8080
+              keycloak:
+                central:
```

Comment:
do we need to add the useAuthTrail here and in line 128?

Reply:
good point! still for now I'd just leave it in the default values file, I may move it into the env specific config as part of https://github.com/eclipse-tractusx/portal-cd/pull/84